lib/travis/build/script/common_lisp.rb

```diff
@@ -0,0 +1,65 @@
+require 'tempfile'
+
+module Travis
+  module Build
+    class Script
+      class CommonLisp < Script
+        DEFAULTS = {
+          lisp: 'sbcl'
+        }
+
+        ROS_URL = "https://raw.githubusercontent.com/snmsts/roswell/release/scripts/install-for-ci.sh"
+
+        SYSTEM_MISSING = %Q[\
+No test system or test script provided.
+Please either override the script: key
+or use the system: key to specify a test ASDF system.
+]
+
+        def configure
+          super
+          sh.cmd 'sudo apt-get update'
+          sh.cmd 'sudo apt-get install libc6:i386 libc6-dev libc6-dev-i386 libffi-dev libffi-dev:i386'
```

Comment:
https://github.com/luismbo/cl-travis/blob/master/install.sh#L31

Don't we need `dpkg --add-architecture` here?